### DISCUSSION TITLE
always return 403 in testing

### DISCUSSION THREAD

**Discussion by razisek** [filamentphp/filament#3007] [Help] [answered]
### Package

Other

### Package Version

^2.0

### Laravel Version

^9.11

### Livewire Version

^2.10

### PHP Version

^8.0.2

### Bug description

i'm make testing in resource like this

```php
it('can view all categories', function () {
    $this->actingAs(User::factory()->withRole(UserRole::SuperAdmin)->create());
    $this->get(MenuCategoryResource::getUrl('index'))->assertSuccessful();
});
```

and the policy viewAny set to ```true```

but, when i do test it always get 403 response
![image](https://user-images.githubusercontent.com/33994563/176984346-f35112d5-61e1-4c6b-a0ef-d5ec2b5c2a1e.png)

**Comment by shankhadevpadam** [comment]
I also got the same issue.

**Comment by pxlrbt** [comment]
I just tested this with the Filament Demo and had no issues.

Did you set up your user model properly as stated here: https://filamentphp.com/docs/2.x/admin/users#authorizing-access-to-the-admin-panel

**Comment by danharrin** [comment, marked as answer]
In this instance, @shankhadevpadam had implemented the FilamentUser interface in their model but it was not returning true, hence the 403.
